lib/src/page_list_viewport_gestures.dart

```diff
@@ -427,11 +555,77 @@ class DeprecatedPanAndScaleVelocityTracker {
     }
 
     final translationDistance = (_lastFocalPosition - _startFocalPosition).distance;
-    if (translationDistance > kViewportReAdjustmentMinTranslationDistance &&
-        velocity.distance < kViewportReAdjustmentMaxVelocity) {
-      // The user was readjusting the viewport by dragging it to the
-      // new position.
+    final velocityDistance = velocity.distance;
```

Comment:
"velocityDistance" -> "speed"

Reply:
Not always. Just trying to avoid the ambiguity using the word speed. If the user scrolls up and down, the speed of the gesture is 0 but instantaneous speed or the absolute value of velocity is velocity.distance. Maybe absoluteVelocity?